cf-agent/cf-agent.c

```diff
@@ -1447,6 +1499,7 @@ static PromiseResult KeepAgentPromise(EvalContext *ctx, const Promise *pp, ARG_U
         if (result != PROMISE_RESULT_FAIL)
         {
             Log(LOG_LEVEL_VERBOSE, "V:     Computing value of \"%s\"", pp->promiser);
+            LogVariableValue(ctx, pp);
```

Comment:
I would put a check for `LogGetGlobalLevel() >= LOG_LEVEL_VERBOSE` here to avoid the runtime cost of constructing the output string when it's not going to be printed.